src/components/AllocationAddForm.js

```diff
@@ -200,12 +200,12 @@ const AllocationAddForm = (props) => {
     const [allocationTypes, setAllocationTypes] = useState([1, 0])
     const [contributorAllocations, setContributorAllocations] = useState(null)
     const [contributorRates, setContributorRates] = useState(null)
-    const [endDate, setEndDate] = useState(moment().add(1, 'months').endOf('month')['_d'])
+    const [endDate, setEndDate] = useState(moment().add(1, 'months').endOf('month'))
     const [mostRecentAllocation, setMostRecentAllocation] = useState(null)
     const [newAllocationRate, setNewAllocationRate] = useState({})
     const [newAllocation, setNewAllocation] = useState({})
     const [rateCurrency, setRateCurrency] = useState(null)
-    const [startDate, setStartDate] = useState(moment().add(1, 'months').startOf('month')['_d'])
+    const [startDate, setStartDate] = useState(moment().add(1, 'months').startOf('month').utc())
```

Comment:
@sofiaromorales double check whether this .utc is needed and if it should also be applied to `endDate`